parlai/utils/bpe.py

```diff
@@ -127,7 +129,7 @@ def add_cmdline_args(argparser):
             '--hf-skip-special-tokens',
             hidden=True,
             type='bool',
-            default=True,
+            default=False,
```

Comment:
Hmm will this default change break compatibility with anything?

Reply:
Adding a warning about this is what's left in the PR.